src/components/configEditor/LabeledInput.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Input, InlineFormLabel } from '@grafana/ui';
+
+interface LabeledInputProps {
+  label: string;
+  tooltip?: string;
+  placeholder?: string;
+  disabled?: boolean;
+  value: string;
+  onChange: (value: string) => void;
+}
+
+export function LabeledInput(props: LabeledInputProps) {
+  const { label, tooltip, placeholder, disabled, value, onChange } = props;
+
+  return (
+    <div className="gf-form">
+      <InlineFormLabel width={12} className="query-keyword" tooltip={tooltip || label}>
+        {label}
+      </InlineFormLabel>
+      <Input
+        disabled={disabled}
+        width={30}
+        value={value}
+        onChange={e => onChange(e.currentTarget.value)}
+        placeholder={placeholder || label.toLowerCase().replace(/ /g, '_')}
```

Comment:
1. why do we need this `replace` call?
2. shouldn't we apply the same logic to the provided input if it's needed?